tests/models/superpoint/test_image_processing_superpoint.py

```diff
@@ -110,3 +132,33 @@ def test_input_image_properly_converted_to_grayscale(self):
         pre_processed_images = image_processor.preprocess(image_inputs)
         for image in pre_processed_images["pixel_values"]:
             self.assertTrue(np.all(image[0, ...] == image[1, ...]) and np.all(image[1, ...] == image[2, ...]))
+
+    @slow
```

Comment:
Why mark as slow? It should be pretty fast

Reply:
This is a leftover from when I used SuperPoint model to produce the keypoint detection output. I removed it